src/main/resources/testrunner/schemas/SUMMARY_testRun.json

```diff
@@ -127,5 +127,583 @@
     "mode": "REPEATED",
     "name": "testScriptResultSummaries",
     "type": "RECORD"
+  },
+  {
+    "mode": "NULLABLE",
```

Comment:
Is wise to mark all field `NULLABLE`? Wouldn't a SQL query "fails" (no data) when some required values are null?

Reply:
The properties marked as `REQUIRED` will never be null because the `TestRunner` framework always generates these properties. The streaming inserts won't happen if any of the required values are null without throwing an exception which we will be able to detect from cloud log.

```
  {
    "mode": "REQUIRED",
    "name": "id",
    "type": "STRING"
  },
  {
    "mode": "REQUIRED",
    "name": "startTime",
    "type": "INTEGER"
  },
```